src/storage/reaper.rs

```diff
@@ -164,13 +164,15 @@ impl<S: Store + Clone + Send + Sync + 'static> Undertaker<S> {
             actors
                 .into_iter()
                 .filter_map(|(id, mut actor)| {
-                    let current_num_hosts = actor.count.len();
+                    let current_num_hosts = actor.instances.len();
                     // Only keep the instances where the host exists
-                    actor.count.retain(|host_id, _| hosts.contains_key(host_id));
+                    actor
```

Comment:
This stuff reminded me that we probably need a `NodeReaped` event so we can trigger reconciliation operations

Reply:
Created #80 to follow up on this